prefs.js

```diff
@@ -183,8 +186,8 @@ const App = new Lang.Class({
                 let inputWidget = input;
 
                 if (input instanceof Gtk.Switch) {
-                    inputWidget = new Gtk.HBox();
-                    inputWidget.pack_end(input, false, false, 0);
+                    inputWidget = new Gtk.Box();
+                    inputWidget.append(input);
```

Comment:
Append would not work in 3.38, either you should check first that is it valid, so that you could use pack_start for 3.38, or else you should remove 3.38 support.

Reply:
Should be fixed